### PR TITLE
delay slides

### LINKED ISSUE #59: Slides: Introduction to epidemiological delays and how to represent them with probability distributions

### PR DESCRIPTION
**Pull request by sbfnk**
Closes #59

### CODE REVIEW

File: sessions/slides/introduction-to-epidemiological-delays.qmd [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+*infection* to *symptom onset*
+
+. . .
+
+**Incubation period**
+
+## Some delays have names
+
+*infection* to *becoming infectious*
```
Comment (seabbs) [inline sessions/slides/introduction-to-epidemiological-delays.qmd:55]: Isn't the definition symptoms not infectiousness?

File: sessions/slides/introduction-to-epidemiological-delays.qmd [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+- Intricate relationship with nowcasting/forecasting
+
+## Quantifying delays
+
+- Epidemiological delays are *variable*
+- We can capture their variability using probability distributions
+
+## Warning: Two levels of uncertainty {background-color="#000000" transition="fade-in" .smaller}
+
+- Probability distributions characterise uncertainty in the delays
```
Comment (seabbs) [inline sessions/slides/introduction-to-epidemiological-delays.qmd:124]: between individuals

**Review by seabbs — COMMENTED** [review]
This is nice. A few minor comments

File: sessions/slides/introduction-to-epidemiological-delays.qmd (hunk 2 above)
Comment (sbfnk) [reply sessions/slides/introduction-to-epidemiological-delays.qmd]: 0383dfb7e8760b8182082b9321e25f8b539a903c

**Review by seabbs — APPROVED** [review]

**Pull request merged**